Data/Sys/GameSettings/RGW.ini

```diff
@@ -22,3 +22,5 @@ SafeTextureCacheColorSamples = 0
 # Fixes visible lines in air vents/fans.
 VertexRounding = True
 ImmediateXFBEnable = False
+# Fixes rabbid tattoo textures saving as black.
+EFBToTextureEnable = False
```

Comment:
Please re-add the newlines to the ends of these two files.

Reply:
Fixed.Oops